provisioning/provisioning-device-client-samples/provisioning-certificate-issuance-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ProvisioningCertificateIssuanceSample.java

```diff
@@ -37,18 +39,25 @@
 @SuppressWarnings("CommentedOutCode") // Ignored in samples as we use these comments to show other options.
 public class ProvisioningCertificateIssuanceSample
 {
-    private static final String idScope = "[Your ID scope here]";
-    private static final String globalEndpoint = "[Your Provisioning Service Global Endpoint here]";
-    private static final ProvisioningDeviceClientTransportProtocol PROVISIONING_DEVICE_CLIENT_TRANSPORT_PROTOCOL = ProvisioningDeviceClientTransportProtocol.HTTPS;
+    private static final String idScope = "";
+    private static final String globalEndpoint = "";
```

Comment:
```suggestion
    private static final String globalEndpoint = "global.azure-devices-provisioning.net";
```

This default value works for everyone who isn't on a govt. or private cloud instance, so we may as well fill it in for them